scripts/SetBaseline.ps1

```diff
@@ -1 +1 @@
-dotnet test --filter "TestCategory=Baseline" -- 'TestRunParameters.Parameter(name=\"SetBaseLine\", value=\"true\")'
```

Comment:
Does this work on Windows? I found that there were some syntactic differences when I first set this up, which is why we also have `scripts/set_baseline.sh`, and this differentiation: https://github.com/Azure/bicep/blob/664e81adfefcfd232ac9dae66b0d2289ab44653e/src/Bicep.Core.UnitTests/Assertions/BaselineHelper.cs#L81-L104

Might have just been regular shell, or older powershell however...

Reply:
I'm not sure what changed, but the existing script started failing for me on Windows recently:

![image](https://user-images.githubusercontent.com/705500/205693871-9d33e116-e03a-4b4c-ae53-7e3ac54ef808.png)

I think it might have been a Powershell upgrade?

Reply:
From what I can tell, Powershell 7.3 requires the version included in this PR, and earlier versions require the version in `main`. I'll back this change out of this PR as it's not germane to tuple types.